python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import json
+import logging
+from typing import Dict, List, Optional, Union
+
+import aiohttp
+from pydantic import HttpUrl
+
+from semantic_kernel.connectors.ai.ai_request_settings import AIRequestSettings
+from semantic_kernel.connectors.ai.ai_service_client_base import AIServiceClientBase
+from semantic_kernel.connectors.ai.chat_completion_client_base import ChatCompletionClientBase
+from semantic_kernel.connectors.ai.ollama.ollama_request_settings import OllamaChatRequestSettings
+from semantic_kernel.connectors.ai.text_completion_client_base import (
+    TextCompletionClientBase,
+)
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+class OllamaChatCompletion(TextCompletionClientBase, ChatCompletionClientBase, AIServiceClientBase):
```

Comment:
This is true for all connectors, since conceptually a text completion and chat completion are the same, it makes things easier when you can also do a text completion with a chat model.